tools/Customization/DevHome.Customization/ViewModels/FileExplorerViewModel.cs

```diff
@@ -152,4 +124,51 @@ public bool EndTaskOnTaskBarEnabled
             _shellSettings.SetEndTaskOnTaskBarEnabled(value);
         }
     }
+
+    public async void AddFolderButton_ClickAsync(object sender, RoutedEventArgs e)
+    {
+        if (experimentationService.IsFeatureEnabled("FileExplorerSourceControlIntegration"))
+        {
+            await Task.Run(async () =>
+            {
+                using var folderDialog = new WindowOpenFolderDialog();
+                var repoRootfolder = await folderDialog.ShowAsync(Application.Current.GetService<WindowEx>());
+                if (repoRootfolder != null && repoRootfolder.Path.Length > 0)
+                {
+                    log.Information($"Selected '{repoRootfolder.Path}' as location to register");
+                    RepoTracker.AddRepositoryPath(unassigned, repoRootfolder.Path);
```

Comment:
What is the significance of using an empty guid?  Does an empty guid signal something else in the program to behave differently?

Reply:
It signals that the user added a repository for tracking in Dev Home but has not yet assigned a source control provider for obtaining metadata to send to File Explorer.

Reply:
I suggest to make your reply the comment to unassigned.  This way future maintainers will know the significance of unassigned.

Reply:
Now that I've gone through and understood how you are using CLSID unassigned makes more sense.  As long as unassigned is used only in this file, I think you don't need to add comments.  But,. if other parts of the code has to deal with a potential empty CLSID, I suggest adding comments.